install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -2491,6 +2491,10 @@
 	Database::query("INSERT INTO `panel_settings` SET `settinggroup` = 'system', `varname` = 'ssl_cipher_list', `value` = 'ECDHE-RSA-AES128-SHA256:AES128-GCM-SHA256:RC4:HIGH:!MD5:!aNULL:!EDH'");
 	lastStepStatus(0);
 
+        showUpdateStep("Adding ssl-protocols setting");
```

Comment:
At the end of the file; also increase db_version in /lib/version.inc.php (YYYYMMDDC) and in install/froxlor.sql

Just check other Updates, but what you did is just for people <0.9.30